delta/testkit/src/main/scala/ch/epfl/bluebrain/nexus/testkit/TestHelpers.scala

```diff
@@ -84,8 +89,8 @@ trait TestHelpers {
     * @return the content of the referenced resource as a json value
     */
   @SuppressWarnings(Array("TryGet"))
-  final def jsonContentOf(resourcePath: String, replacements: Map[String, String]): Json =
-    parse(contentOf(resourcePath, replacements)).toTry.get
+  final def jsonContentOf(resourcePath: String, attributes: Map[String, Any]): Json =
```

Comment:
We can add another jsonContentOf

```scala
final def jsonContentOf(resourcePath: String, attributes: (String, Any)*): Json =
    jsonContentOf(resourcePath, attributes.toMap)
```

And refactor in the places we use `jsonContentOf` to use this instead. Honestly wrapping it into a Map doesn't add any extra value :)

Reply:
In most cases yes but sometimes when we reuse common attributes, we can concat maps for example

Reply:
I have added the second method though so we can use it too when needed